pkg/pillar/hypervisor/kvm.go

```diff
@@ -271,6 +271,7 @@ const qemuDiskTemplate = `
 {{- else}}
 [device "vhost-disk{{.DiskID}}"]
   driver = "vhost-scsi-pci"
+  max_sectors = "16384"
```

Comment:
Hey @giggsoff -- sorry to come late to this -- but can you include the same comment you included in the PR description in the big comment block at the top of `kvm.go`? Just submit a quick PR and we can merge it very easily.